### PR TITLE
Add Unit Testing for Components and Reducers

### LINKED ISSUE #41: Add Unit Testing

### PR DESCRIPTION
**Pull request by DanielEFrampton**
### Sidebar Checklist
​
- [x] Ran test suite
- [x] Request reviewers
- [x] Assign yourself and other contributors
- [x] Link to project
- [x] Added to Readme
​
### Issues Resolved
Resolves #41 
​
### Problem Addressed
Test coverage still below requirements on rubric.
​
### Solution Implemented
Added simple unit testing for all reducers and components.
​
### Screenshots (optional)
​

### Other Notes

### CODE REVIEW

**Comment by DanielEFrampton** [comment]
Screen Shot 2020-04-14 at 8 58 17 PM

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import '../../styles/Navbar.scss';
 
-const Navbar = () => {
+export const Navbar = () => {
```
Comment (jobannon) [inline src/components/Navbar/Navbar.jsx:4]: needed for test coverage - as discussed

**Review by mintona — APPROVED** [review]
Yes! Strong work getting these tests up and running.

**Comment by DanielEFrampton** [comment]
After excluding `src/index.js` and `src/reducers/index.js`: Screen Shot 2020-04-14 at 9 02 29 PM

**Review by jobannon — APPROVED** [review]

**Pull request merged**